lib/plugins/globalid.js

```diff
@@ -0,0 +1,49 @@
+/**
+ * Treasure Global ID
+ */
+
+// Modules
+var jsonp = require('jsonp')
+var _ = require('../utils/lodash')
+var invariant = require('invariant')
+var cookie = require('cookies-js')
+
+function cacheSuccess (result, cookieName) {
+  cookie.set(cookieName, result['global_id'], { expires: 6000 })
+  return result['global_id']
+}
+
+function configure () {
+}
+
+function fetchGlobalID (success, error, forceFetch) {
```

Comment:
I think forceFetch should be the first argument, having it at the end seems weird